### DISCUSSION TITLE
is it sensible to add node_modules subdir to NuxtConfig

### DISCUSSION THREAD

**Discussion by danielmittank** [nuxt/nuxt#20994] [Questions]
Hello,

I'm new to nuxt, and still have to understand some concepts.

In my Vue2 Apps i use a library of own components, for example Buttons, InputFields and so on.

Installed via npm install they are imported and registered in my main.js.

Now, with auto-imports in nuxt3 i just wanna know if I should just add the component-directory (for example ~/node_modules/myButton) to the components-dirs in my NuxtConfig.

**Comment by maou-shonen** [comment]
It should depend on your preference. You can also refer to this section in the docs.
https://nuxt.com/docs/guide/directory-structure/components#library-authors
